server/src/controllers/client_timeline_entries/addClientTimelineEntry.js

```diff
@@ -0,0 +1,94 @@
+const logger = require("pino")();
+const ClientTimelineEntry = require("../../models/client_timeline_entry.model");
+const {
+  submitPlacementUpdateEntryInTimelines,
+} = require("../../utils/placement_entry_util");
+
+const addClientTimelineEntryRequestHandler = async (req, res) => {
```

Comment:
When creating a new timeline entry, we should be generating the title and body from info passed in through the request rather than relying on the frontend to generate the title and body entirely

Reply:
Hi @DanielDervishi, I was planning on doing the auto generation on the timeline routing ticket, just so that when I work on connecting the frontend I can picture all the possible entries I need to autogenerate from in the frontend, it will be hard to do it now with purely / take some time to consider all the needed cases now to auto generate. Was hoping to focus this in the timeline routing ticket to make sure I cover all the possible timeline entry creation. I hope this is fine.

Reply:
Hey Kevin, that works as well